master/internal/config/config_test.go

```diff
@@ -899,76 +905,46 @@ func TestPickVariation(t *testing.T) {
 		expected    string
 	}{
 		{
-			name: "Light Horizontal prioritized",
-			variations: MediaAssetVariations{
-				LightHorizontal: "light-horizontal",
-				LightVeritical:  "light-vertical",
-				DarkHorizontal:  "dark-horizontal",
-				DarkVeritical:   "dark-vertical",
-			},
+			name:        "Light Horizontal prioritized",
```

Comment:
```suggestion
			name:        "Light Horizontal prioritized no mode",
```

Reply:
didn't need the test names in the first place. I took em out